python/pyspark/tests/test_taskcontext.py

```diff
@@ -127,12 +127,12 @@ def f(iterator):
 
         def context_barrier(x):
             tc = BarrierTaskContext.get()
-            time.sleep(random.randint(1, 10))
+            time.sleep(random.randint(1, 5) * 2)
```

Comment:
This seems fine but if you're just looking for longer delays, randint(5,10) or something?

Reply:
Maybe i had to add a comment from the original PR .. it has actually a reason for that https://github.com/apache/spark/pull/30277#discussion_r519038459